Sources/NIOHPACK/HPACKHeader.swift

```diff
@@ -461,7 +502,8 @@ extension String.UTF8View {
     ///
     /// - Parameter bytes: The string constant in the form of a collection of `UInt8`
     /// - Returns: Whether the collection contains **EXACTLY** this array or no, but by ignoring case.
-    fileprivate func compareCaseInsensitiveASCIIBytes(to other: String.UTF8View) -> Bool {
+    @inlinable
```

Comment:
I've not read the rules but I don't see any logical reason why something shouldn't be both inlineable and fileprivate.

Reply:
Totally reasonable except that the rule is it has to be `internal` or higher.